src/site/xdoc/security.xml

```diff
@@ -49,9 +49,9 @@ limitations under the License.
             <subsection name="Security Model">
                 <p>
                     The <a href="https://commons.apache.org/security.html#Security_Model">Apache Commons security model</a>
-                    specifies that it is unsafe to pass possibly malicious input to Commons libraries.
-                    For Commons Configuration 2.x, loading untrusted configuration files or performing operations
-                    on them should not allow code execution, and should not cause any denial of service situations.
+                    specifies that it is unsafe to pass possibly-malicious input to Commons libraries.
```

Comment:
There is no hyphen needed.

Reply:
it read a bit easier that way to me, but I'm fine with dropping it ;)